ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -858,25 +975,12 @@ public void shouldStreamInserts() throws Exception {
     assertThat(acksPublisher.isFailed(), is(false));
   }
 
-  @Test
-  public void shouldHandleErrorResponseFromStreamInserts() {
```

Comment:
As above: this is important test coverage (that error responses are properly exposed to the user). If we are removing the specific cause of the error used in the existing test, then let's replace the error with a different error thrown from the endpoint, rather than removing the test entirely.

Reply:
Mentioned above, I can add more case in different PR

Reply:
As above: https://github.com/confluentinc/ksql/pull/8114#discussion_r709545064